query/Undefiner.java

```diff
@@ -141,9 +142,7 @@ else if (variable.valueType().isPresent()) {
 
     private ThingType getThingType(LabelConstraint label) {
         try (ThreadTrace ignored = traceOnThread(TRACE_PREFIX + "gettype")) {
-            ThingType thingType;
-            if ((thingType = conceptMgr.getThingType(label.label())) != null) return thingType;
-            else return null;
```

Comment:
the concept manager already returns a null when the label doesn't exist, so this check is redundant